docs/pages/admin-guides/access-controls/access-requests/resource-requests.mdx

```diff
@@ -406,6 +406,72 @@ Requesting access to a Kubernetes Namespace allows you to access all resources
 in that namespace but you won't be able to access any other supported resources
 in the cluster.
 
+##### Enforce users to request only allowed Kubernetes Resources
+
+The `request.kubernetes_resources` field allows you to restrict what kinds of Kubernetes 
+resources a user can access request to.
+
+The following role assigned to a user, restricts access requesting to only Kubernetes namespaces.
+If a user attempts to make a request to Kubernetes resources other than `namespace` then the request
+will be rejected:
```

Comment:
The grammar feels a bit off, maybe this is better
```suggestion
The following role, when assigned to a user, would allow them to request access to Kubernetes namespaces.
Requests for Kubernetes resources other than `namespace` will not be allowed.
```